src/send/utils.ts

```diff
@@ -115,3 +115,7 @@ export function* handlePaymentDeeplink(deeplink: string) {
     Logger.warn('handlePaymentDeepLink', `deeplink ${deeplink} failed with ${e}`)
   }
 }
+
+export function hasAddressField(obj: Object): obj is { address: string } {
```

Comment:
there is a helper function recipientHasAddress in src/recipients/recipient.ts that could be used instead of this

Reply:
Jean alerted me to it awhile back when he noticed I added hasAddressField somewhere else (looks like you moved it here?) but I was hustling to fix a bug at the time